nfcreader/build.gradle

```diff
@@ -8,10 +21,16 @@ android {
         minSdkVersion 19
         targetSdkVersion 29
         versionCode 1
-        versionName "1.0"
+        versionName getVersionName()
```

Comment:
What is the need of this function, the version name is available directly through `BuildConfig.VERSION_NAME` as you used it

Reply:
This is going to dynamically fetch the tag or version released on github. without us to manually update it each time. 
getVersionName() is used for getting dynamic tag / version released on Github

Reply:
Okay if so I think you need to provide a default value if there is no tag to be fetched in the catch block instead of the null

Reply:
yes sure